jcstress/.gitignore

```diff
@@ -0,0 +1,6 @@
+target/
+libs/
+results/
+*.bin.gz
+test.iml
+src/main/java/komem/litmus/
```

Comment:
why this line in `.gitignore`?

Reply:
These are files generated by `jcstress-wrapper`. I couldn't be bothered to change Maven project layout...

Reply:
Can you add `jcstress/generated` somewhere on the path to emphasis that this folder is related to jcstress and is automatically generated?

Reply:
Done!